### PR TITLE
Note support status in docs

### LINKED ISSUE #379: Clarify support status
Based on akka/akka-grpc#454 when that is done

### PR DESCRIPTION
**Pull request by johanandren**
Fixes #379 

Started out with only the support status, ideas where to add recommended service discovery mechanisms welcome.

### CODE REVIEW

File: docs/src/main/paradox/index.md [hunk 1]
```diff
@@ -14,3 +14,11 @@ Support for some protocols is provided out of the box and others are provided as
...
 
 @@@
+
+
+## Compatibility & support
+
+This project does not yet have to obey the rule of staying binary compatible between releases that is common 
+for Akka libraries. Breaking API changes may be introduced without notice as we refine and simplify based on your feedback.
+
+Akka gRPC is currently *Incubating*. The Lightbend subscription does not yet cover support for this project.
```
Comment (patriknw) [inline docs/src/main/paradox/index.md]: gRPC

**Comment by patriknw** [comment]
fyi @TimMoore

File: docs/src/main/paradox/index.md (hunk 1 above)
Comment (johanandren) [reply docs/src/main/paradox/index.md]: Mmmm that copy pasta

**Comment by TimMoore** [comment]
Maybe https://developer.lightbend.com/docs/akka-management/current/discovery/index.html could be split into "core discovery methods" and "contributed discovery methods". Or we could list the recommended ones in the ["Discovery Method trade-offs"](https://developer.lightbend.com/docs/akka-management/current/discovery/index.html#discovery-method-trade-offs) section. That page is a bit long, to be honest, so maybe it would be a good idea to break it up into sub-pages with an index that designates which ones are recommended/Lightbend-maintained.

**Comment by chbatey** [comment]
Yep, we can do that in another PR and get this one in

**Pull request merged**